plugins/auth-backend/api-report.md

```diff
@@ -480,6 +493,23 @@ export const providers: Readonly<{
       userIdMatchingUserEntityAnnotation(): SignInResolver<OAuthResult>;
     }>;
   }>;
+  bitbucketServer: Readonly<{
+    create: (
+      options?:
+        | {
+            authHandler?: AuthHandler<BitbucketServerOAuthResult> | undefined;
+            signIn?:
+              | {
+                  resolver: SignInResolver<BitbucketServerOAuthResult>;
+                }
+              | undefined;
+          }
+        | undefined,
+    ) => AuthProviderFactory;
+    resolvers: Readonly<{
+      emailMatchingUserEntityProfileEmail: () => SignInResolver<unknown>;
```

Comment:
I thought this one was going to say `SignInResolver<BitbucketServerOAuthResult>`.

Reply:
Should/can I do something about this?

Reply:
you can declare the type explicitly like github does here https://github.com/backstage/backstage/blob/01a075ec1df4d493b94df660ecc5660fd0837898/plugins/auth-backend/src/providers/github/provider.ts#L331